### DISCUSSION TITLE
wait-for-airflow-migrations

### DISCUSSION THREAD

**Discussion by Dobrunyya** [apache/airflow#29425] [Q&A]
Good afternoon, I'm collecting a helm chart and trying to disable db migration, I registered everywhere

waitForMigrations:
     enabled: false

but there is no such parameter in the worker deployment and therefore it waits.
Please tell me how to get rid of this error

**Comment by potiuk** [comment]
Look there: https://airflow.apache.org/docs/helm-chart/stable/index.html#installing-the-chart-with-argo-cd-flux-or-terraform

**Comment by potiuk** [comment]
But your problem description does not tell what your problem is, so this might be a bad answer to not well asked question as well.

**Comment by Dobrunyya** [comment]
I disabled job db migration, and set it in all deployments
   waitForMigrations:
     enabled: false

and these deployments start successfully, but the worker does not have such a velus, which is why it is waiting for migration, please tell me how to remove the waiting for migration from the worker

**Reply by potiuk** [reply]
No idea. Your description is pretty vague and contains no examples/resource generated,no details on what exactly you've done.i suggest you to provide the detailed example and code snippets:

* Which helm chart you are using in which version 
* What you've done as configuration
* Commands you run to install helm
* What problems you observe (log output) 
* Ideally relevant output of helm's --dry-run resources that show what has been generated
* What you expected to be generated 


I simply do not understand what exactly you've done and what problem you have. It might be easier to help you if you spend a little time on detailing your problem
